src/app/access-requests/features/access-request-manager-dialog/access-request-manager-dialog.component.ts

```diff
@@ -68,15 +84,36 @@ export class AccessRequestManagerDialogComponent implements OnInit {
   #confirmationService = inject(ConfirmationService);
   #notificationService = inject(NotificationService);
 
+  #configService = inject(ConfigService);
+  helpdeskUrl = this.#configService.helpdesk_url;
+
   #ivas = this.#ivaService.userIvas;
   ivas = this.#ivas.value;
   ivasAreLoading = this.#ivas.isLoading;
   ivasError = this.#ivas.error;
 
   #ivaTypePipe = inject(IvaTypePipe);
 
+  editNotesAndTicket = inject(AccessRequestFieldEditService);
+
   selectedIvaIdRadioButton = model<string | undefined>(undefined);
 
+  editTicketId: Signal<EditableFieldInfo> = signal({
```

Comment:
This should be a WritableSignal, since we need to update the value. Actually, since this is also passed to a subcomponent, it should be probably a ModelSignal, i.e. created as a model().